inference-engine/src/vpu/graph_transformer/include/vpu/frontend/frontend.hpp

```diff
@@ -213,6 +215,7 @@ class FrontEnd final {
 
     using LayerParser = std::function<void(const Model&, const ie::CNNLayerPtr&, const DataVector&, const DataVector&)>;
     const ie::details::caseless_map<std::string, LayerParser> parsers;
+    const ie::details::caseless_map<std::string, LayerParser> activationParsers;
```

Comment:
@nikita-kud what do you think about encapsulating these parsers inside the activation parser? In order not to introduce such a new structures which may confuse someone with the difference between this and other parsers.